src/sql/base/browser/ui/table/plugins/checkboxSelectColumn.plugin.ts

```diff
@@ -129,23 +136,8 @@ export class CheckboxSelectColumn<T extends Slick.SlickData> implements Slick.Pl
 		this._grid.invalidateRow(row);
 		this._grid.render();
 		this._grid.setActiveCell(row, this.index);
-		this.checkSelectAll();
```

Comment:
you have moved the select all logic to table.component.ts, that means other places using this plugin directly won't get the same feature.

Reply:
Check all logic is still available. I have moved the select rows logic to table component as the 2 select row logics in RowSelectionModel plugin and selectCheckBox plugin were not interacting well with each other.